apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/commons/QueryUtilsTest.java

```diff
@@ -128,6 +132,60 @@ public void testCreateDateRangePredicateBothBounds() {
     verify(builder).and(notNullPredicate, lowerBoundPredicate, upperBoundPredicate);
   }
 
+  /** Test {@link QueryUtils#hasClaimsData} testing claims bitmask values. */
```

Comment:
Can we make this a little more descriptive please? (Describe the test to someone reading this in 6-12 months)

Reply:
Done.